apps/projects/app/components/Content/IssueDetail/DetailsCard.js

```diff
@@ -42,69 +45,49 @@ const DetailsCard = ({ issue }) => {
     grid: PropTypes.string.isRequired,
   }
 
-  const OpenedBy = ({ author }) => (
-    <Text size="large" css="display: flex; align-items: center">
-      <img src={author.avatarUrl} alt="user avatar" css="margin: 8px; width: 24px; border-radius: 50%;" />
-      <Link
-        href={author.url}
-        target="_blank"
-        style={{ textDecoration: 'none', color: `${theme.link}` }}
-      >
-        {author.login}
-      </Link>
-    </Text>
-  )
-  OpenedBy.propTypes = userGitHubShape
-
   const SummaryTable = ({ issue }) => (
     <div css={`
       display: grid;
-      grid-template-columns: repeat(${issue.hasBounty ? '3' : '4'}, 1fr);
+      grid-template-columns: repeat(${issue.hasBounty ? '2' : '1'}, 1fr);
```

Comment:
Do you think it would be cleaner/clearer if we did something like this?

```js
margin-top: ${1.5 * GU}px;
${issue.hasBounty && css`
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-template-rows: auto;
  grid-template-areas:
    'deadline exp'
    'description description'
  ;
  grid-gap: ${3 * GU}px;
  align-items: stretch;
`}

Reply:
Non-funded before & after:
![Screenshot from 2019-11-01 14-53-05](https://user-images.githubusercontent.com/34452131/68030324-814d8000-fcb9-11e9-9782-f28b451f29a7.png)
![Screenshot from 2019-11-01 15-06-31](https://user-images.githubusercontent.com/34452131/68030328-85799d80-fcb9-11e9-9633-0b93dfd6ba08.png)

Funded before & after:
![Screenshot from 2019-11-01 14-54-30](https://user-images.githubusercontent.com/34452131/68030350-90ccc900-fcb9-11e9-89e9-b40fd10a3508.png)
![Screenshot from 2019-11-01 15-07-15](https://user-images.githubusercontent.com/34452131/68030352-92968c80-fcb9-11e9-9bb7-bf100f39237a.png)

Reply:
I've also improved that CSS to be cleaner.